.github/ISSUE_TEMPLATE/support-general-requirement.md

```diff
@@ -8,11 +8,15 @@ assignees: ''
 ---
 
 **Is your support request related to a problem? Please describe.**  
-A clear and concise description of what the problem is. Ex. I have problems with [...]
+<!-- A clear and concise description of what the problem is. Ex. I have problems with [...] -->
 
 **Describe the solution you'd like**  
-A clear and concise description of what you want to happen.
+<!-- A clear and concise description of what you want to happen. -->
 
 **Additional context**  
-Add any other context or screenshots about the feature request here.  
-Link: [https://your/url](https://your/url)  
+<!-- 
+Add any other context or screenshots about the feature request here.
+Link: [https://your/url](https://your/url)
+-->
+  
```

Comment:
Maybe unnecessary blank lines, but not essential

Reply:
https://github.com/eclipse-tractusx/sig-infra/pull/23/commits/958ed6e927a4fc141ff6821027504c508cb3494d